test/deployment/resources/resource_request_test.go

```diff
@@ -35,23 +35,23 @@ var _ = ginkgo.Describe("ResourceRequestsTests", func() {
 		deployBroker(brokerDeployer)
 		pod := getPod(ctx1)
 		actualMemRequest := pod.Spec.Containers[0].Resources.Requests.Memory()
-		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemLimit, actualMemLimit.String())
+		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemRequest, actualMemRequest.String())
 	})
 
-	ginkgo.It("Memory Request check", func() {
+	ginkgo.It("Memory Request check_Update", func() {
 		expectedMemRequest := "512M"
 		brokerDeployer.WithMemRequest(expectedMemRequest)
 		deployBroker(brokerDeployer)
 		pod := getPod(ctx1)
 
 		actualMemRequest := pod.Spec.Containers[0].Resources.Requests.Memory()
-		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemLimit, actualMemLimit.String())
+		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemRequest, actualMemRequest.String())
```

Comment:
I think this one was already fixed by older PR and not sure why GH still show it. 
Here is the current line from main.
https://github.com/artemiscloud/activemq-artemis-operator-test-suite/blob/d9f278bd9b4eb7a34fab44a5a582333cea13ca6c/test/deployment/resources/resource_request_test.go#L48